examples/sendToAndTransferFromStealthAddress/components/stealth-actions-example.tsx

```diff
@@ -21,7 +21,7 @@ import 'viem/window';
 import { useBalance, useWaitForTransactionReceipt } from 'wagmi';
 
 const StealthActionsExample = () => {
-  const rpcUrl = import.meta.env.VITE_RPC_URL;
+  const rpcUrl = import.meta.env.VITE_RPC_URL || 'https://1rpc.io/sepolia	'; // Use your rpc url here; fallback to 1rpc.io for Sepolia
```

Comment:
still showing me that VITE_RPC_URL is required
<img width="340" alt="image" src="https://github.com/user-attachments/assets/dd9dc262-17f2-4bf8-bd6b-e937957c46fa">

Reply:
https://github.com/ScopeLift/stealth-address-sdk/pull/77/commits/e1935ebfad4792e74f6b1f13f666961c60204a43